pos_access_right/static/src/js/pos_access_right.js

```diff
@@ -205,7 +201,6 @@ odoo.define("pos_access_right.pos_access_right", function (require) {
          * Block 'Payment' button if user doesn't belong to the correct group
          */
         renderElement: function () {
-            var self = this;
```

Comment:
Hi. I don't understand how the line ``self.gui.get_user_groups();`` can work if you remove this line. Did I missed something ?

Reply:
@legalsylvain  hello,
  use this.gui.get_user_groups(); instade of self.gui.get_user_groups();